jenkinsapi/node.py

```diff
@@ -124,9 +125,10 @@ def get_node_attributes(self):
 
             retries = na['max_num_retries'] if 'max_num_retries' in na else ''
             re_wait = na['retry_wait_time'] if 'retry_wait_time' in na else ''
+            sshslave_class = 'com.cloudbees.jenkins.plugins.sshslaves.SSHLauncher' if na['connector'] == 'Cloudbees' else 'hudson.plugins.sshslaves.SSHLauncher'
```

Comment:
This will fail if "connector" is not in `na`. 

For compatibility it is better to replace that line with:

```
connector = na.get('connector', 'hudson.plugins.sshslaves.SSHLauncher')
sshslave_class = 'com.cloudbees.jenkins.plugins.sshslaves.SSHLauncher' if connector == 'Cloudbees' else connector
```